lib/ansible/modules/cloud/ovirt/ovirt_storage_domains.py

```diff
@@ -99,6 +99,7 @@
             - "C(lun_id) - LUN id."
             - "C(username) - A CHAP user name for logging into a target."
             - "C(password) - A CHAP password for logging into a target."
+            - "C(override_luns) - If I(True) storage domain will be formatted before adding it."
```

Comment:
Maybe it would be nice to say, something like:
If `True` ISCSI storage domain luns will be overriden before adding.
What do you think?